test/test-file.cpp

```diff
@@ -46,38 +46,41 @@ namespace quick_lint_js {
 namespace {
 filesystem::path make_temporary_directory();
 void write_file(filesystem::path, const std::string &content);
+filesystem::path temp_file_path;
 
 class test_file : public ::testing::Test {
  public:
   filesystem::path make_temporary_directory() {
-    // TODO(strager): Delete the directory when the test finishes.
     return quick_lint_js::make_temporary_directory();
   }
+
+ protected:
+  void TearDown() override { filesystem::remove_all(temp_file_path); }
 };
 
 TEST_F(test_file, read_regular_file) {
-  filesystem::path temp_file_path =
-      this->make_temporary_directory() / "temp.js";
-  write_file(temp_file_path, "hello\nworld!\n");
+  temp_file_path = this->make_temporary_directory();
```

Comment:
Assigning to `temp_file_path` everywhere is cumbersome and error-prone. Instead of assigning to `temp_file_path` here, can we assign to `temp_file_path` inside the `make_temporary_directory` function (on line 54)?